leaderboards/leaderboard_challenges.tex

```diff
@@ -0,0 +1,160 @@
+    \label{fig:kick_leaderboard}
+\end{figure}
+\subsubsection{Challenge Execution}
+Teams will be given 3 attempts at kicking to each of the 3 target postitions (illustrated in \cref{fig:kick_leaderboard}):
+\begin{itemize}
+    \item The centre mark in the centre circle
+    \item The penalty mark on the opposite side of the field
+    \item A corner on the opposite side of the field
+\end{itemize}
+In each attempt, participating teams must start the robot behaviour of their competing robot via a single chest button
+press. The robot then has \qty{\PenaltyShootoutKickTime}{\second} to kick the ball towards the current target.
+The competing robot must not touch the ball a second time after the ball has clearly moved,
+otherwise the attempt ends immediately without scoring.
+Once the ball has come to a complete stop, the distance from the ball to the target is measured.
+
+Code changes are only allowed when changing targets.
+\subsubsection{Scoring}
+The sum of the smallest distance from each target will be recorded in the leaderboard.
+Distances are rounded to the closest centimetre.
+
+\subsection{Most Passes Leaderboard}
+This leaderboard measures the highest number of successful passes a team can
+complete in a given timeframe, emphasizing passing accuracy, speed, and coordination.
+
+\subsubsection{Setup}
+This leaderboard challenge will take place on a standard SPL field and requires 2 competing robots
+and 1 standard SPL soccer ball.
+
+Competing robots will be placed on the touchlines on opposite sides of the same half of the field,
+in line with the penalty mark. The ball will be placed on the corner of the penalty area.
+
+\begin{figure}[t]
+    \centerline{\includegraphics[width=\columnwidth]{figs/passing_leaderboard.pdf}}
+    \caption{Passing Kick Leaderboard: Robots completing the attempt are shown in blue, target area is shown in green, exclusion area is shown in magents and ball placement is shown in red}
+    \label{fig:passing_leaderboard}
+\end{figure}
+\subsubsection{Challenge Execution}
+In each attempt, participating teams must start the robot behaviour of their competing robot via a single chest button
+press. Teams will then have \qty{\MaxTimePassingLeaderboard}{\sec} to move the ball from the starting
+position to the target area while completing as many passes as possible. The centre
+1.5 metres of the field (Centre Cricle Diameter) is an exclusion zone, no robots are allowed within this area,
+if a robot enters this area, or the ball comes to a complete stop
+within this area, the run is ended immediately. If the ball is kicked off the field,
+it is placed back on the touchline where it crossed. Each run is considered complete
+once a robot has posession of the ball within the target area.
+
+\subsubsection{Scoring}
+1 point is awarded every time the ball crosses the exlusion zone, with an additional point
+awarded if the ball touches the recieving robot before it stops rolling.
```

Comment:
Rather than being required to touch the robot, could we define an area in front of the robot similar to the passing challenge of 2021. The reason for this is that in real football, a good pass is played slightly in front of a player (in the direction they are moving), not directly at their feet.